elements/pfe-styles/src/pfe-type.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
```

Comment:
I'm not sure the name of this file is clear.  Type has a lot of loaded meanings in web dev.  Can we be more explicit and call it `pfe-typography`?

Reply:
ah, that's a good reason. yep I can update